### PR TITLE
Update configuring-user-provisioning-for-your-enterprise.md

### PR DESCRIPTION
**Pull request by cpratap994**
Here the tenant url is working only while we append /api/v3/scim/v2

### Why:
Here the tenant url is working only while we append /api/v3/scim/v2

### What's being changed:
appended /api/v3 in sample tenant url.

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
@cpratap994 Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Comment by hubwriter** [comment]
Related pull request: https://github.com/github/docs/pull/3175

File: content/admin/authentication/configuring-user-provisioning-for-your-enterprise.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ You must have administrative access on your IdP to configure the application for
 
   | Value | Other names | Description | Example |
   | :- | :- | :- | :- |
-  | URL | Tenant URL | URL to the SCIM provisioning API for your enterprise on {% data variables.product.prodname_ghe_managed %} | <code>https://<em>YOUR-GITHUB-AE-HOSTNAME</em>/scim/v2</code> |
+  | URL | Tenant URL | URL to the SCIM provisioning API for your enterprise on {% data variables.product.prodname_ghe_managed %} | <code>https://<em>YOUR-GITHUB-AE-HOSTNAME</em>/api/v3/scim/v2</code> |
```
Comment (alanhalim) [inline content/admin/authentication/configuring-user-provisioning-for-your-enterprise.md:77]:
```suggestion
  | URL | Tenant URL | URL to the SCIM provisioning API for your enterprise on {% data variables.product.prodname_ghe_managed %} | <code>https://<em>YOUR-GITHUB-AE-HOSTNAME</em>/api/v3/scim/v2</code> |
```

**Review by alanhalim — APPROVED** [review]
Thanks

**Comment by megbird** [comment]
👋  @cpratap994 thank you so much for the suggestion! Right now, I can't verify that this is correct. You should check in with our awesome [support team](https://support.github.com/contact) and let us know if you’d like a docs update after that!

**Comment by Nguyenvanduong98ht** [comment]
> Here the tenant url is working only while we append /api/v3/scim/v2
> 
> ### Why:
> Here the tenant url is working only while we append /api/v3/scim/v2
> 
> ### What's being changed:
> appended /api/v3 in sample tenant url.
> 
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [ ]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [ ]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Comment by Nguyenvanduong98ht** [comment]
9921be8

**Comment by Nguyenvanduong98ht** [comment]
{"type":"module"}

**Pull request closed without merging**